cmd/capabilities/utils/utils.go

```diff
@@ -0,0 +1,6 @@
+package utils
```

Comment:
I think this is good advice about "utils" packages: https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
Planning to refactor in a separate PR. Tracking issue: https://github.com/apigee/registry/issues/100

Reply:
SGTM - thanks!